schema.yaml

```diff
@@ -11,179 +11,18 @@ properties:
       type: NAMESPACE
```

Comment:
I am wondering whether we should use the [V2 of the schema API](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/master/docs/schema.md#v2-specification). It has some fields that are required and do not appear in v1. Just being paranoid here about reasons they might reject the listing.